pkg/external-provider/provider.go

```diff
@@ -51,13 +52,15 @@ func (p *externalPrometheusProvider) GetExternalMetric(ctx context.Context, name
 		return nil, provider.NewMetricNotFoundError(p.selectGroupResource(namespace), info.Metric)
 	}
 	// Here is where we're making the query, need to be before here xD
-	queryResults, err := p.promClient.Query(ctx, pmodel.Now(), selector)
+	queryResults, queryErr := p.promClient.Query(ctx, pmodel.Now(), selector)
 
-	if err != nil {
-		klog.Errorf("unable to fetch metrics from prometheus: %v", err)
+	if queryErr != nil {
+		klog.Errorf("unable to fetch metrics from prometheus: %v", queryErr)
```

Comment:
Can you insert `external` in between `fetch` and `metrics`? An improvement to more easily disambiguate where these are coming from